wallet-server-ui/src/app/component/splash/splash.component.ts

```diff
@@ -1,44 +1,62 @@
-import { animate, animateChild, query, style, transition, trigger } from '@angular/animations';
-import { Component, OnInit } from '@angular/core';
+import {
+  animate,
+  animateChild,
+  query,
+  style,
+  transition,
+  trigger,
+} from '@angular/animations';
+import { Router } from '@angular/router';
 
+import { Component, OnInit } from '@angular/core';
 
-const SPLASH_KEY = 'noSplash'
+const SPLASH_KEY = 'noSplash';
 
 @Component({
   selector: 'splash',
   templateUrl: './splash.component.html',
   animations: [
-  // the fade-in/fade-out animation.
-  trigger('fadeOut', [
-    transition(':leave', [
-      query(':leave', animateChild(), {optional: true}),
-      animate(300, style({opacity: 0}))
+    // the fade-in/fade-out animation.
+    trigger('fadeOut', [
+      transition(':leave', [
+        query(':leave', animateChild(), { optional: true }),
+        animate(300, style({ opacity: 0 })),
+      ]),
     ]),
-  ]),
-],
-  styleUrls: ['./splash.component.scss']
+  ],
+  styleUrls: ['./splash.component.scss'],
 })
 export class SplashComponent implements OnInit {
+  showSplash = false;
+  dontShowSplashAgain = false;
 
-  showSplash = false
-
-  constructor() { }
+  constructor(private router: Router) {}
 
   ngOnInit(): void {
     // Force reset splash key
     // localStorage.removeItem(SPLASH_KEY)
 
-    const show = localStorage.getItem(SPLASH_KEY) === null
-    this.showSplash = show
+    const show = localStorage.getItem(SPLASH_KEY) === null;
+    this.showSplash = show;
   }
 
-  dontShowSplashAgainClick() {
-    console.debug('dontShowSplashAgainClick()')
-    localStorage.setItem(SPLASH_KEY, '1')
+  onStartOnboardingClicked() {
+    this.saveSplashSettings();
+    this.showSplash = false;
+
+    this.router.navigate(['/onboarding']);
   }
 
-  onClick() {
-    this.showSplash = !this.showSplash
+  onSkipOnboardingClicked() {
+    this.saveSplashSettings();
+    this.showSplash = false;
+
+    this.router.navigate(['/wallet']);
   }
 
+  saveSplashSettings() {
+    if (this.dontShowSplashAgain) {
+      localStorage.setItem(SPLASH_KEY, '1');
+    }
+  }
 }
```

Comment:
I'm open to other ways of structuring the logic here, but as a first pass, I added two action buttons to the splash screen UI — “onboarding” and “skip onboarding” — and converted the “Don’t show this screen again” button to a checkbox. 

From there, I'm just saving the checkbox value in local storage depending on how it’s set when users click on one of the action buttons.